esmvalcore/cmor/_fixes/shared.py

```diff
@@ -72,9 +72,7 @@ def add_plev_from_altitude(cube):
             pressure_bounds = altitude_to_pressure(height_coord.core_bounds())
         pressure_coord = iris.coords.AuxCoord(pressure_points,
                                               bounds=pressure_bounds,
-                                              var_name='plev',
```

Comment:
Can you briefly explain why this needs to be changed? I'm don't immediately see how this is connected to the scalar coordinates.

Reply:
Without these changes, iris complains about differing coordinates:
```
iris.exceptions.MergeError: failed to merge into a single cube.
  Coordinates in cube.dim_coords differ: air_pressure.
```
The same problems occurs when converting to altitude instead of pressure levels. This error can be reproduced with e.g. this test recipe: [recipe_test.yml.txt](https://github.com/ESMValGroup/ESMValCore/files/8013680/recipe_test.yml.txt)